vigetlabs/wordpress-site-starter
# [#32] Composer Scripts

# Summary

This PR adds Composer Scripts that configure a new project. This *should* work if you download the `.zip` or run the composer create-project command (but that won't work until this PR is merged).

This is the first iteration so we can get it up on packagist.org.

## Issues

* #32 

## Testing Instructions

1. Download the `.zip` of the `bd/32-composer-scripts`
2. Run `composer install`
3. Follow the prompts

What should happen:

1. The files should get updated with all the strings
2. WordPress should be installed
3. The site should open in a browser (`ddev launch`)
